sarsen/scene.py

```diff
@@ -48,17 +48,17 @@ def transform_dem_3d(
         dem_3d.sel({dim: 1}).values.flat,
         dem_3d.sel({dim: 2}).values.flat,
     )
-    dem_3d_crs: xr.DataArray = xr.zeros_like(dem_3d)
+    dem_3d_crs: xr.DataArray = xr.zeros_like(dem_3d)  # type: ignore
```

Comment:
```suggestion
    dem_3d_crs: xr.DataArray = xr.zeros_like(dem_3d)
```
Fixed here: https://github.com/pydata/xarray/pull/6630